caproto/_dbr.py

```diff
@@ -40,6 +40,12 @@
 class_name_t = 40 * ctypes.c_char  # epicsOldString
 
 
+def epics_timestamp_to_unix(secondsSinceEpoch, nanoSeconds):
+    '''UNIX timestamp (seconds) from Epics TimeStamp structure'''
+    return (EPICS2UNIX_EPOCH + secondsSinceEpoch + 1.e-6 *
```

Comment:
@danielballan @klauer Assistance is here! Sorry for the delay, we were busy this sunday fighting fires on aisle 2.

![image](https://cloud.githubusercontent.com/assets/8185425/23629482/bb20a5d6-026c-11e7-8a21-72874506574a.png)